swan-lake/development-tutorials/deployment-guide/virtual-machine-deployment.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: ballerina-cloud-left-nav-pages-swanlake
+title: Virtual machine deployment
+description: If you prefer to execute Ballerina services directly on virtual machines (VMs) or bare metal servers without using containerization, you can follow these steps to set up and run your services in a production environment, 
+keywords: ballerina, programming language, services, cloud, kubernetes, docker
+active: ballerina-deployment-guideLines
+intro: If you prefer to execute Ballerina services directly on virtual machines (VMs) or bare metal servers without using containerization, you can follow these steps to set up and run your services in a production environment, 
+---
+
+1. Prepare the Server Environment.
+    - Provision the VM or Bare Metal Server,
+      - Ensure the server meets the hardware requirements for your application (CPU, memory, disk space, etc.).
+      - Configure the server OS (Linux is recommended for production).
+
+2. Install the Ballerina distribution from [the Ballerina Installation options.](https://ballerina.io/downloads/)
+
+3. Install any other dependencies your Ballerina service might need (e.g., database drivers, libraries).
+
+4. Write Ballerina code.
+      **service.bal**
```

Comment:
<img width="873" alt="image" src="https://github.com/user-attachments/assets/895517e2-5584-48d0-8128-ee0479af644d" />
Check the above